python/run_fccanalysis.py

```diff
@@ -494,7 +494,8 @@ def apply_filepath_rewrites(filepath: str) -> str:
 def run_local(config: dict[str, any],
               args: object,
               analysis: object,
-              infile_list):
+              infile_list,
+              do_weighted = False):
```

Comment:
There is a dictionary `config` which holds variables which were merged between script file and command line arguments

Reply:
Added the `do_weighted` there, thanks!